Fixed Quickstart (#1621)
near/docs: docs/2.develop/contracts/quickstart.md

1. The `get_greeting` method is a [`view`](./anatomy.md#public-methods) method, meaning it only reads from the contract and can be called for free by anyone.
2. By default, the contract is initialized with the `greeting` attribute set to `"Hello"`.
3. The `set_greeting` method is a [`change`](./anatomy.md#public-methods) method, meaning it modifies the contract's state and requires a user to sign a transaction in order to be executed.

---

## Build and Test

Building and testing the contract is as simple as running two commands.

<Tabs>
  <TabItem value="🌐 JavaScript">

  ```bash
  npm run build
  npm run test

  # Expected:
  # returns the default greeting ✅
  # changes the greeting ✅
  ```

  </TabItem>
  <TabItem value="🦀 Rust">

  ```bash
  ./build.sh
  ./test.sh

  # Expected:
  # Passed ✅ gets default greeting
  # Passed ✅ changes the greeting
  ```
  
  </TabItem>
</Tabs>

<details>
<summary> Failing tests? </summary>

If the tests are failing, make sure that you are using `node v16` and the `toolchain v1.69` in `rust`. You can always use

- `nvm use 16` to switch to `node v16`
- `rustup default 1.68` to switch to `toolchain v1.69`

</details>

> In the background these commands are calling the build tools for each language, and invoking the [Sandbox](/develop/testing/integration-test) tests from the `sandbox-ts/rs` directory.

Reviewer: ```suggestion
In the background, these commands are calling the build tools for each language and invoking the [Sandbox](../testing/integration.md) tests from the `sandbox-ts/rs` directory.
```

:::tip Sandbox

> Testing the contracts within a Sandbox allows us to understand how the contract will behave once deployed to the network, while having total control over the testing environment.

Reviewer: ```suggestion
Testing the contracts within a Sandbox allows you to understand how the contract will behave once deployed to the network while having total control over the testing environment.
```

```

<details>
<summary> Example Result </summary>

```bash
New account "lovely-event.testnet" created successfully.
```

</details>

:::tip
Here we are creating a random account since we do not care about the account's name. Remember that you can create a named account through any wallet (i.e. [MyNearWallet](https://testnet.mynearwallet.com)) and then use it from the `near-cli` by invoking `near login`.
:::

---

## Deploy the Contract
Having our account created, we can now deploy the contract into it:

<Tabs>
  <TabItem value="🌐 JavaScript">

  ```bash
  near deploy --wasmFile build/release/hello.wasm --accountId <created-account>
  ```

  </TabItem>
  <TabItem value="🦀 Rust">

  ```bash
  near deploy --wasmFile ./target/wasm32-unknown-unknown/release/hello_near.wasm --accountId <created-account>
  ```
  
  </TabItem>
</Tabs>

**Congrats**! your contract now lives in the NEAR testnet network.

---

## Interacting with the Contract

To interact with your deployed smart contract you can call its methods using the `near-cli` tool.

#### Get Greeting

> The `get_greeting` method is a [`view`](./anatomy.md#public-methods) method, meaning it only reads from the contract's state, and thus can be call for **free**.

Reviewer: ```suggestion
The `get_greeting` method is a [`view`](./anatomy.md#public-methods) method, meaning it only reads from the contract's state, and thus can be called for **free**.
```
